xspec-dev/testing/xspec-basic.xspec

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="../xspec-assurance.sch"?>
+<x:description xmlns:mx="http://csrc.nist.gov/ns/csd/metaschema-xslt"
+   xmlns="http://example.com/ns/computer"
+   xmlns:x="http://www.jenitennison.com/xslt/xspec" xmlns:xs="http://www.w3.org/2001/XMLSchema"
+   stylesheet="copy_me.xsl" xslt-version="3.0">
```

Comment:
This file and `xspec-dev/testing/test/xspec-basicR.xspec` differ only in their relative path to the stylesheet being tested. Did you mean to have the file in both places, such as because they are illustrating different pieces of the infrastructure? Did you consider having one file use `<x:import>` to get the global variable and scenarios from the other file?

Reply:
I think this is a relic of wishing to test Saxon's recursive retrieval - I will remove.